django_model_changes/changes.py

```diff
@@ -148,7 +148,11 @@ def old_state(self):
         return self._states[0]
 
     def _changes(self, other, current):
-        return dict([(key, (was, current[key])) for key, was in other.items() if was != current[key]])
+        changes = {}
+        for key, was in other.items():
+            if key in current and was != current[key]:
```

Comment:
note: the `key in current` check is the new behavior here. I thought this list comprehension was getting a bit confusing